pages/tutorials/excel-sheet-into-python-dashboard.mdx

```diff
@@ -0,0 +1,157 @@
+---
+ogTitle: Turn excel sheet into Python dashboard
+description: How to create an interactive dashboard with Python using data from an Excel sheet in Jupyter Notebook. 
+ogImage: tutorials
+---
+
+import {Screenshot} from 'components/screenshot';
+import excelSheet from "../../public/images/tutorials/dashboards/excel-dashboard/excel-sheet.png";
+
+# Turn excel sheet into Python dashboard
+
+We created a simple Python dashboard using data from an Excel sheet and `pandas`, `mercury` and `altair` packages. 
+In this article you will learn:
+
+- how to load excel sheets to your Jupyter Notebook,
+- how to use Markdown, NumberBox, Range and Table `mercury` widgets,
+- how to create charts using `vega-altair`,
+- how to turn your notebook into Web App.
+
+
+## Import required packages
+
+You will need following packages: 
+
+```python
+pandas 
+mercury
+altair
+```
+
+- `pandas`  is  for data manipulation and analysis.
+- `mercury` will be used for interactive widgets and for running notebook as a web app.
+- `altair` for creating declarative statistical visualizations.
+
+Make sure that you have installed them before start doing anything.
+
+The cell code with imports :
+
+```python
+import pandas as pd 
+import mercury as mr 
+import altair as alt
+```
+
+## Get Data 
+
+In this case, we used an example excel sheet:
+
+<Screenshot src={excelSheet} alt="Example excel sheet." />
+
+You need to load the data from an Excel file into a DataFrame. This will allow to manipulate and analyze the data easily.
+
+```python 
+df = pd.read_excel('example.xlsx')
+```
+
+## Create Dashboard
+
+### Set heading 
+Please set a heading for the dashboard using the [**Markdown**](/docs/output-widgets/markdown/) widget:
+
+```python
+mr.Markdown(text="# **Employee Hours Log**")
+```
+
+### Create boxes with numbers
+
+To provide a quick overview of the **Salary** and **Wage** data, please create two sets of [**NumbeBox**](/docs/output-widgets/numberbox/) widgets. The first set will show the lowest, highest, and average salaries, and the second set will show the same statistics for wages:
+
+```python 
+mr.NumberBox([
+    mr.NumberBox(data=f"{df['Salary'].min()}$", title="Lowest salary"),
+    mr.NumberBox(data=f"{df['Salary'].max()}$", title="Highest salary"),
+    mr.NumberBox(data=f"{round(df['Salary'].mean())}$", title="Average salary")
+])
+```
+
+```python 
+mr.NumberBox([
+    mr.NumberBox(data=f"{df['Wage'].min()}$", title="Lowest wage"),
+    mr.NumberBox(data=f"{df['Wage'].max()}$", title="Highest wage"),
+    mr.NumberBox(data=f"{round(df['Wage'].mean())}$", title="Average wage")
+])
+```
+
+### Add interactive widget
+
+To allow users to filter the data based on age range, you need to include an interactive [**Range**](/docs/input-widgets/range/) widget. This will let users select an age range and dynamically update the visualizations based on their selection.
```

Comment:
maybe:

Please include an interactive [**Range**](/docs/input-widgets/range/) widget to allow filter the data based on age range.